.circleci/real_config.yml

```diff
@@ -1513,14 +1513,15 @@ jobs:
 
   test-e2e-react:
     docker:
-      - image: cimg/node:20.9.0
+      - image: mcr.microsoft.com/playwright:v1.40.0-focal
     resource_class: large
     steps:
       - checkout-with-sm
       - skip-if-docs-only
       - skip-if-github-only
+      - run: apt-get update && apt-get -y install make
       - react-get-deps
-      - run: npx playwright install --with-deps chrome firefox webkit
+      - run: npx playwright install --with-deps && npx playwright install chrome
```

Comment:
Did you mean to install Firefox and webkit in the first install command? Is that installing nothing, or is there a default file with stuff to install somewhere?

Reply:
[This doc](https://playwright.dev/docs/ci#introduction) might be helpful. 
We list browsers here 
https://github.com/determined-ai/determined/blob/d8ed945b073c250f4d598b8eaef3b81b847b16fc/webui/react/playwright.config.ts#L26-L40
and  `npx playwright install --with-deps && npx playwright install chrome` install these dependencies somewhere.